tests/audio/test_pesq.py

```diff
@@ -75,25 +75,28 @@ def average_metric(preds, target, metric_func):
 class TestPESQ(MetricTester):
     atol = 1e-2
 
-    @pytest.mark.parametrize("ddp", [True, False])
+    @pytest.mark.parametrize(MetricTesterDDPCases.name_strategy(), MetricTesterDDPCases.cases_strategy())
```

Comment:
I would say it is cleaner if it will be a constant or function returning tuple:
```py
def ddp_name_options():
    return "ddp", (True, False)
```
and later:
```py
@pytest.mark.parametrize(*ddp_name_options())
```

Reply:
but anyway why is this needed?

Reply:
The reason I added this stategy was that now all available test cases are generated and you can explicitly test them locally. This way you can guarantee full coverage, don't depend on the ci machines or some launch flags and each test states on which device it runs. This way you dont have to check the machine stats to e.g. find out, if a test failed on single or multi gpu.
In order to optimize the tests, having an additional skip condition with an env var could achieve the same. Especially for test optimization you could then run ddp=false on a machine with a single gpu and ddp=True on one with multiple.
I'm fine putting it into a single function, my idea was to stick with two parameter style of `parametrize`

Reply:
I agree with @Borda that it would be better to  return a tuple there.

For the general purpose: I think we should split this between arguments for every test (like devices and maybe also ddp strategies) for which we should use pytest fixtures (that's what they are there for) and then you could call `pytest --devices=XX tests`. For test-specific arguments this approach is fine IMO

Reply:
I'm unsure how to continue with this. Any suggestions?